### PR TITLE
gocb 2.6.1 update

### PR DESCRIPTION
**Pull request by henesgokdag**
i can't fix opentelemetry_test.go because metrictest is removed

### CODE REVIEW

File: metrics.go [hunk 1]
```diff
@@ -32,11 +33,7 @@ func (meter *OpenTelemetryMeter) Counter(name string, tags map[string]string) (g
 	meter.lock.Lock()
 	counter := meter.counterCache[key]
 	if counter == nil {
-		otCounter, err := meter.wrapped.NewInt64Counter(name)
-		if err != nil {
```
Comment (chvck) [inline metrics.go:36]: Why the removal of the error handling?

**Comment by chvck** [comment]
Thanks @henesgokdag sorry for the slow review. I'll try to find some time to look into how to update testing now that metrictest doesn't exist.

File: metrics.go (hunk 1 above)
Comment (henesgokdag) [reply metrics.go:36]: that's my mistake I missed error handling and now I added it again

Comment (henesgokdag) [reply metrics.go:36]: @chvck

**Comment by chvck** [comment]
Hi @henesgokdag apologies for being slow on this. Ideally I'd like to see the metrics tests fixed before we merge but I haven't had time to look into it yet.

**Comment by chvck** [comment]
The other thing is that we will need you to sign our CLA before we can accept the PR, the process is detailed at https://github.com/couchbase/couchbase-jvm-clients/blob/master/CONTRIBUTING.md#pull-requests - whilst this project isn't in Gerrit it's still how we do CLA signing.

**Comment by chvck** [comment]
Closing as we have now merged a change to update to 2.7.2

**Pull request closed without merging**